Example/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../tsconfig.json",
+  "baseUrl": ".",
+  "include": ["**/*.ts", "**/*.tsx", "**/*.js"]
```

Comment:
`include` already [contains ts, tsx, and js extensions](https://www.typescriptlang.org/tsconfig#include)

> If a glob pattern doesn’t include a file extension, then only files with supported extensions are included (e.g. .ts, .tsx, and .d.ts by default, with .js and .jsx if allowJs is set to true).

So you can change this to `"include": ["**/*"]`, given you set `allowJs: true`.